lib/cucumber/runtime.rb

```diff
@@ -229,6 +229,7 @@ def filters
         filters << Cucumber::Core::Test::LocationsFilter.new(filespecs.locations)
         filters << Filters::Randomizer.new(@configuration.seed) if @configuration.randomize?
         filters << Filters::Quit.new
+        filters << Filters::Retry.new(@configuration)
```

Comment:
Should `Quit` be the last filter, so that it is possible to abort the retry of a long running test case?